### PR TITLE
toolchain: Fix generating robots.txt file DOCS-455

### PR DESCRIPTION
**Pull request by prcr**
The changes from the previous pull request https://github.com/codacy/docs/pull/1578 didn't work because we were creating the `robots.txt` file in the wrong directory. The idea is to create the file inside the directory `./site` where all the generated HTML files are so that it's pushed together with them to the branch `gh-pages`.

Besides that, I had copied Pulse's domain name by mistake! :sweat_smile:

### CODE REVIEW

**Review by nicklem — APPROVED** [review]
I should pay more attention when checking code edits...

**Review by claudiacarpinteiro — APPROVED** [review]
I'll wait for this merge before I merge [#173](https://github.com/codacy/pulse-user-docs/pull/173) 😉

**Comment by prcr** [comment]
> I should pay more attention when checking code edits...

What counts is that we can fail fast! :rocket:

By the way, the fact that the domain name was wrong was what allowed me to understand that the previous version wasn't working because the domain didn't change in [the commit created by running the workflow](https://github.com/codacy/docs/commit/3053915fa2a8fb66e48efe276a9fcf5afe8f2df2). :laughing:

**Pull request merged**
